docs/manual/execution-lifecycle/job-data.md

```diff
@@ -0,0 +1,202 @@
+# Job Data Plugins (Enterprise) [Incubating]
+::: enterprise
+:::
+
+::: incubating
+:::
+
+This plugin allows Jobs to export a JSON file as the result of an execution.  The JSON data can be retrieved via the API, and shown in the GUI.
+
+This allows you to run a Job, and receive a JSON result back, customized to contain data that is relevant to the outcome of the Job.
+
+There are currently two different Plugins that can be used to produce JSON data for the Job.
+
+## Requirements
+
+::: warning
+You must enable the feature with the following configuration
```

Comment:
To formalize our docs we've been avoiding using 'You' in
"To use this feature enable it with the following setting"

We can also now point to the Configuration Management setup for feature flags rather than the text based Rundeck Config.

Reply:
I will make some changes